### PR TITLE
[Chore] Folder structure update

### PR DESCRIPTION
**Pull request by Akinmyde**
__What does this PR do?__
This PR re-structure the folder for the App

__Description of Task to be completed?__
* `Use babel-node in development only and node in production`
* `Create a server folder and place other folders inside the server folder except for the test folder`
* `Move all babel package from dependencies to Devdependencies`

__How should this be manually tested?__
`Clone the repo`
`Checkout to branch<folder-structure>`
`Take a look at the folders`

__Screenshots__
Screenshot 2019-03-13 at 8 59 45 PM

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,4 +3,4 @@ node_modules/
 .coveralls.yml
 .nyc_output/
 coverage/
-.coveralls.yml
\ No newline at end of file
+dist/
```
Comment (sojida) [inline .gitignore]: Great Work, I think you should consider EOLs here

File: package.json [hunk 2]
```diff
@@ -21,16 +22,18 @@
   },
   "homepage": "https://github.com/Akinmyde/library-backend#readme",
   "devDependencies": {
+    "@babel/cli": "^7.2.3",
+    "@babel/core": "^7.3.4",
+    "@babel/node": "^7.2.2",
+    "@babel/polyfill": "^7.2.5",
+    "@babel/preset-env": "^7.3.4",
     "babel-register": "^6.26.0",
```
Comment (sojida) [inline package.json]: Nice job on the Babel update. Are we still using `babel-register`? If not please uninstall/remove from package.json

**Comment by McHardex** [comment]
Nice job. The updates came very fast however I would like to know if you created a PULL_REQUEST_TEMPLATE.MD file for the PR template. If No, I would suggest that you make one so that all team members can have a consistent PR template for our subsequent PRs and if Yes, kudos already!!!

File: .gitignore (hunk 1 above)
Comment (Akinmyde) [reply .gitignore]: What do you mean by that?

File: package.json (hunk 2 above)
Comment (Akinmyde) [reply package.json]: we will be needing babel-register for testing with mocha

**Comment by Akinmyde** [comment]
> Nice job. The updates came very fast however I would like to know if you created a PULL_REQUEST_TEMPLATE.MD file for the PR template. If No, I would suggest that you make one so that all team members can have a consistent PR template for our subsequent PRs and if Yes, kudos already!!!

**Comment by Akinmyde** [comment]
@McHardex I followed the PR template shared with us on slack on Tuesday

File: .gitignore (hunk 1 above)
Comment (sojida) [reply .gitignore]: End of Line

**Comment by AnayoOleru** [comment]
LGTM

**Comment by walsamlee** [comment]
We are good

**Pull request merged**